src/toolcontext.nit

```diff
@@ -58,6 +58,13 @@ class Message
 	# * enclose identifiers, keywords and pieces of code with back-quotes.
 	var text: String
 
+	# The severity level
+	#
+	# * 0 is advices (see `ToolContext::advice`)
+	# * 1 is warnings (see `ToolContext::warning`)
+	# * 2 is errors (see `ToolContext::error`)
+	var level: Int
```

Comment:
For now, I think you should do it like with any language without enums: use symbolic constants. :)